5.0/en/0x10-V1-Architecture.md

```diff
@@ -155,6 +155,12 @@ This is a placeholder for future documentation requirements.
 | **1.14.6** | [MOVED TO 50.7.2] | | | | |
 | **1.14.7** | [MODIFIED, MOVED FROM 1.1.5] Verify that all communication needs for the application are documented. This should include external services which the application relies upon and cases where an end user might be able to provide an external location to which the application will then connect. | | ✓ | ✓ | 1059 |
 
+## V1.50 Web Frontend Security Documentation
+
+| # | Description | L1 | L2 | L3 | CWE |
+| :---: | :--- | :---: | :---: | :---: | :---: |
+| **1.50.1** | [ADDED] Verify that application documentation states the expected security features that browsers using the application should support (such as HTTPS, HSTS, Content Security Policy (CSP), and other relevant HTTP security mechanisms). It should also define how the application must behave when such an event happens (such as warning the user or blocking access). | | | ✓ | |
```

Comment:
Actually "such an event happens" doesn't work here (because we did not tell which event we are talking about). This should be something such as:

> It should also define how the application must behave when some of these features are not available (such as warning the user or blocking access)